conda_build/build.py

```diff
@@ -347,17 +347,17 @@ def build(m, get_src=True, verbose=True, post=None):
             f.write(u'\n'.join(sorted(list(files1))))
             f.write(u'\n')
 
+        script = m.get_value('build/script', None)
+        if isinstance(script, list):
+            script = '\n'.join(script)
```

Comment:
I don't really understand what the difference between using /b and not using /b would be here. I do agree it would be good to add `|| exit 1` here.